src/base/bittorrent/session.cpp

```diff
@@ -3900,6 +3900,11 @@ void Session::startUpTorrents()
         MagnetUri magnetUri;
         int queuePosition;
         if (readFile(fastresumePath, data) && loadTorrentResumeData(data, torrentParams, queuePosition, magnetUri)) {
+            // Make sure the torrent will restored in "paused" state
+            // Then we will start it if needed
+            data.replace("6:pausedi0e", "6:pausedi1e");
```

Comment:
I wonder, why do you propose a hack instead of changing libtorrent to get the desired result?

Reply:
I have yet another (non-hacker) idea, but I'm not sure if it will work. I'll test it soon.